tests/foreman/api/test_reporttemplates.py

```diff
@@ -16,105 +16,62 @@
 :Upstream: No
 """
 
-from robottelo.decorators import tier1, tier2, stubbed
+from robottelo.decorators import tier1, tier2, stubbed, skip_if_bug_open
```

Comment:
Since #7404 was merged, there is no `skip_if_bug_open` anymore - we have `skip_if` now.